src/sentry/models/organizationslugreservationreplica.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import BoundedBigIntegerField, Model, region_silo_only_model, sane_repr
+from sentry.services.hybrid_cloud import REGION_NAME_LENGTH
+
+
+@region_silo_only_model
+class OrganizationSlugReservationReplica(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    slug = models.SlugField(unique=True, db_index=True)
+    organization_id = BoundedBigIntegerField(db_index=True)
```

Comment:
Should this be a `FlexibleForeignKey`? as `Organization` and `OrganizationSlugReservationReplica` are co-located.

Reply:
In this case, probably not. Because the organization slug reservation can precede the existence of the organization during provisioning, making this a `FlexibleForeignKey` would prevent us from replicating. I'll be adding async deletion + replication tests to verify that we clean these up properly.